lib/parser/ast/processor.rb

```diff
@@ -124,7 +124,16 @@ def process_argument_node(node)
       alias on_kwarg          process_argument_node
       alias on_kwoptarg       process_argument_node
       alias on_kwrestarg      process_argument_node
-      alias on_procarg0       process_argument_node
+
+      def on_procarg0(node)
+        if node.children[0].is_a?(Symbol)
+          # s(:procarg0, :a)
```

Comment:
I think we don't need this branch anymore.

Reply:
But what if the builder is not "modernized"?

Reply:
Then it works just like before.

But I think it's fine to leave this in. Just needs a comment to indicate why this is using an AST format that's not what our docs describe.